.promu.yml

```diff
@@ -0,0 +1,17 @@
+go:
+    cgo: true
+repository:
+    path: github.com/oliver006/redis_exporter
+build:
+    flags: -a -tags 'netgo static_build'
+    ldflags: |
```

Comment:
fixed. There is a problem here, so I set CGO_ENABLE=0 before `promu`manually. See [#60](https://github.com/prometheus/promu/issues/60)